docs/src/quickstart.md

```diff
@@ -160,6 +161,51 @@ plot_energy(idata);
 gcf()
 ```
 
+### Additional information in Turing.jl
+In the above `from_mcmcchains` call, we did not specify the `prior`, `posterior_predictive`, nor the `log_likelihoods`. This section describes how one would go about obtaining these from the `Model` and `Chains`, which we can then pass to `from_mcmcchains` to construct the [`InferenceData`](@ref).
+
+To sample from the prior, one simply calls `sample` but with the `Prior` sampler:
+```@example quickstart
+prior = sample(param_mod, Prior(), nsamples; progress = false)
+```
+Obtaining the predictive posterior is a matter of first instantiating a "predictive model", i.e. a `Model` but with the observations set to `missing`, and then call `predict` on the predictive model and the inferred posterior:
+```@example quickstart
+# Instantiate the predictive model
+param_mod_predict = turing_model(
+    J, fill(missing, length(y)), σ
+)
+# and then sample!
+posterior_predictive = predict(param_mod_predict, turing_chns)
+```
+And to extract the elementwise log-likelihoods, which is useful if you want to compute metrics such as [`loo`](@ref),
+```@example quickstart
+loglikelihoods = elementwise_loglikelihoods(param_mod, turing_chns)
+```
+This can then be included in the `from_mcmcchains` call from above:
+```@example quickstart
+# Convert into format compatible with ArviZ.jl
+using LinearAlgebra
+loglikelihoods_arr = permutedims(cat(values(loglikelihoods)...; dims = 3), (2, 1, 3))
+
+idata = from_mcmcchains(
+    turing_chns,
+    prior = prior,
+    posterior_predictive = posterior_predictive,
+    coords = Dict("school" => schools),
+    dims = Dict(
+        "y" => ["school"],
+        "σ" => ["school"],
+        "θ" => ["school"],
+    ),
+    library = "Turing",
+    log_likelihood = loglikelihoods_arr
+)
+```
+
```

Comment:
Can you just give one sentence introducing LOO?
Also, it would be nice to show a plot after this, maybe
```julia
plot_loo_pit(idata; y = "y", ecdf = true)
```

Reply:
Good idea; I added a couple of sentences + plot